iam.yml

```diff
@@ -0,0 +1,120 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: >-
+  Define IAM Roles utilized to build, deploy, and operate the JavaBuilder service.
+  Note: CAPABILITY_IAM capability is required to deploy this stack.
+
+Parameters:
+  ArtifactBucket:
+    Type: String
+    Description: Name of existing S3 bucket for storing pipeline artifacts
+  TemplateBucket:
+    Type: String
+    Description: Name of existing S3 bucket for storing CloudFormation template artifacts
+
+Resources:
+  JavaBuilderLambdaExecutionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Action: ['sts:AssumeRole']
+            Effect: Allow
+            Principal: {Service: [lambda.amazonaws.com]}
+      Path: /
+      Policies:
+        - PolicyName: JavaBuilderLambdaExecutionPolicy
```

Comment:
NIT: Javabuilder is one word. We're not camelcasing it because it leads to complications in other areas. (i.e. java_builder, java.builder, JavaBuilder, and Java Builder are a bit tough to search for compared to javabuilder and Javabuilder)

Reply:
Done! I definitely support standardizing naming/capitalization.